package.json

```diff
@@ -29,7 +29,8 @@
   "author": "Brainhub",
   "license": "MIT",
   "dependencies": {
-    "express": "^4.15.2"
+    "express": "^4.15.2",
+    "filesize": "^3.6.0"
```

Comment:
This dependency will be included in the bundle. So it's needed only during the build process. That's why it should be in the devDependencies.